tests/RuleSetTest.php

```diff
@@ -516,6 +516,53 @@ public function testGetMissingRuleConfiguration()
         $ruleSet->getRuleConfiguration('_not_exists');
     }
 
+    /**
+     * @dataProvider providePHPUnitMigrationSetDefinitionNameCases
+     *
+     * @param string $setName
+     */
+    public function testPHPUnitMigrationTargetVersions($setName)
+    {
+        $ruleSet = new RuleSet([$setName => true]);
+
+        foreach ($ruleSet->getRules() as $ruleName => $ruleConfig) {
+            if (true === $ruleConfig) {
```

Comment:
it's not readable what we are doing it.
make a private helper?
`$targetVersion = true === $ruleConfig ? $this->getDefaultPHPUnitTargetOfRule($ruleName) : $ruleConfig;`